src/group/group.router.ts

```diff
@@ -14,10 +18,12 @@ import UpdateUserRole from './user/endpoints/group.user.update';
 export default createFeatureRouter(
   new GetGroupByID(),         // GET    /:id
   new GetGroupByUserID(),     // GET    /users/:id
-  new SearchGroupByName(),    // GET    /
+  new SearchGroup(),          // GET    /
   new CreateGroup(),          // POST   /
-  new UpdateGroup(),          // PUT    /:id
+  new UpdateGroup(),          // PATCH  /:id
   new DeleteGroup(),          // DELETE /:id
+  new AddTagToGroup(),        // PUT    /:id/tags
```

Comment:
I think that because the route of remove tag from group has the label, the route of add tag to group should have it too.

Reply:
You are correct. But not only because of the `RemoveTag` URI. In a PUT request the resource should always appear in the request URI.

Reply:
Ok, good to know.